README.md

```diff
@@ -1,92 +1,74 @@
+![](https://raw.githubusercontent.com/NOAA-OWP/owp-open-source-project-template/master/doc/bluecyan.gif)
 
+The project and program contain the following elements. 
+  - **Technology stack**: The hydrofabric pre-processor, river network traversal framework, and time series data model are all written in python. The routing model engines are primarily written in fortran, but we are also experimenting with ML-based methods and can imagine any number of options called from within the network traversal framework.
+  - **Status**:  The project is currently in development phase 2 -- we are making the first connections of the various components within a single module. Phase 3 shoulud begin around July 2020 and we will be working demonstrations of the framework with operational outputs bootstrapped from the current national water model. Eventually, there will be a [CHANGELOG](CHANGELOG.md).
+  - **Demos**: The `notebooks` folder has a number of python notebooks, many of which can be executed from within the Google colaboratory environment, which demonstrate various aspects of the project. 
 
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
+## Configuration and Dependencies
 
-![](https://raw.githubusercontent.com/NOAA-OWP/owp-open-source-project-template/master/doc/Screenshot.png)
-
-
-## Dependencies
+This program uses the following system packages:
+```
+python3
+python3-devel
+gcc-gfortran
+```
 
-Describe any dependencies that must be installed for this software to work.
-This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
-If specific versions of other software are required, or known not to work, call that out.
+... and the following python modules:
+```
+geopandas 
+numpy 
+pandas 
+xarray 
+netcdf4 
+```
 
 ## Installation
 
-Detailed instructions on how to install, configure, and get the project running.
-This should be frequently tested to ensure reliability. Alternatively, link to
-a separate [INSTALL](INSTALL.md) document.
+please see usage and testing below. Standby for docker container instructions in the near future.
 
 ## Configuration
 
-If the software is configurable, describe it in detail, either here or in other documentation to which you link.
+Presently, there are no specific configuration details. Stand by.
 
-## Usage
+## Usage and Testing
+The following should provide a sense of the operation of the routing scheme:
 
-Show users how to use the software.
-Be specific.
-Use appropriate formatting when showing code snippets.
-
-## How to test the software
-
-If the software includes automated tests, detail how to run those tests.
+```
+pip3 install geopandas numpy pandas xarray netcdf4 
+git clone --progress --single-branch --branch fortran http://github.com/jameshalgren/wrf_hydro_nwm_public.git
```

Comment:
If this code is moving to this repo, then this section should reflect this.  Can be done after initial setup though.

Reply:
That's easy enough to update.

Reply:
```suggestion
git clone --progress --single-branch --branch master http://github.com/jameshalgren/t-route.git
```